grpc/oob/bmc/bmc.go

```diff
@@ -207,27 +234,32 @@ func (m Action) UpdateUser(ctx context.Context) error {
 
 // DeleteUser functionality for machines.
 func (m Action) DeleteUser(ctx context.Context) error {
-	timer := prometheus.NewTimer(metrics.ActionDuration.With(prometheus.Labels{"service": "bmc", "action": "Delete_user"}))
+	timer := prometheus.NewTimer(metrics.ActionDuration.With(prometheus.Labels{"service": "bmc", "action": "delete_user"}))
 	defer timer.ObserveDuration()
 
+	tracer := otel.Tracer("pbnj")
+	ctx, span := tracer.Start(ctx, "client.DeleteUser")
+	defer span.End()
+
 	host, user, password, err := m.ParseAuth(m.DeleteUserRequest.Authn)
 	if err != nil {
 		return err
 	}
+	span.SetAttributes(attribute.String("bmc.host", host), attribute.String("bmc.username", user))
 
 	creds := &v1.UserCreds{Username: m.DeleteUserRequest.Username}
 	status := fmt.Sprintf("deleting user %q", creds.GetUsername())
 	m.SendStatusMessage(status)
 
 	actions, err := m.setupConnection(ctx, user, password, host, creds)
 	if err != nil {
-		m.SendStatusMessage("connectiion setup failed")
+		// setupConnection is responsible for sending a status message and updating the span
```

Comment:
don't we still want to send a `m.SendStatusMessage()` here?

Reply:
There's a SendStatusMessage in setupConnection. Seems duplicative to send two messages where one should suffice.

Reply:
I could also move the comment above the function call and consolidate this into a 1 line `if` statement... Thoughts?